packages/react-charting/src/utilities/utilities.ts

```diff
@@ -1090,11 +1090,12 @@ export function rotateXAxisLabels(rotateLabelProps: IRotateLabelProps) {
   return Math.floor(maxHeight / 1.414); // Compute maxHeight/tanInverse(45) to get the vertical height of labels.
 }
 
-export function wrapTextInsideDonut(selectorClass: string, maxWidth: number) {
+export function wrapTextInsideDonut(selectorClass: string, maxWidth: number, val: string) {
   let idx: number = 0;
+
   d3SelectAll(`.${selectorClass}`).each(function () {
     const text = d3Select(this);
```

Comment:
is val equal to text. You are not using text to get the words.

Reply:
yes, it is the value inside donut